src/ql/com/topology.cc

```diff
@@ -7,6 +7,9 @@
 
 #include "ql/utils/logger.h"
 
+// #define MULTI_LINE_LOG_DEBUG to enable multi-line dumping 
+#undef MULTI_LINE_LOG_DEBUG
```

Comment:
No need to undef, since there's an assumption this has file scope. I'll file an issue t extend the logging system with finer control at run time

Reply:
Ok, I'll update this, commit and merge.